### PR TITLE
cseeger/golang post

### PR DESCRIPTION
**Pull request by cseeger-epages**
First Version

### CODE REVIEW

**Comment by BirgitBader** [comment]
@cseeger-epages This is the first review fixing the layout and some general issues.
Please go through the post again and check if you can shorten some stuff here and there. It's a bit too long.
With regards to the code samples, please check this file https://github.com/ePages-de/epages-docs/blob/develop/helpdocs/examples.md > section "Highlights" . I don't know if your examples are shown correctly, see screenshot attached.
bildschirmfoto 2017-07-12 um 09 03 31
With regards to the header image: can we use it like this with the "own drawings"?

**Comment by cseeger-epages** [comment]
i tried to shorten the text a bit but its not that much on the code side it would be hard to understand if i would shorten it more. 
I used the highlight anotation but i dont know if the golang anotation is implemented or supported please feel free to add golang to this syntax highlighting.
For the header image: i used some picture from google and reworked it in colors and using these gopher drawings. From my site its ok but i dont know about copyright stuff. If you have sth better no problem.

**Comment by BirgitBader** [comment]
@cseeger-epages it would be the best if you could get a picture from https://www.pexels.com/ or https://pixabay.com/ as we don't have issues with copyrights there. Thanks :)

File: _posts/2017/2017-08-01-going-forward.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+date: "2017-08-01 08:00:00"
+image: blog-header/golang-library.jpg
+categories: tech-stories
+authors: ["Carsten", "Karsten"]
+---
+
+Long time no see guys.
+In my last blogpost I talked about [pipelines](https://developer.epages.com/blog/2015/08/03/creating-systems-with-pipelining.html).
+Today we take a deeper look into some of the special parts of these pipelines.
+This post deals with an i-doit backend library written in [GO(LANG)](https://golang.org/).
```
Comment (BirgitBader) [inline _posts/2017/2017-08-01-going-forward.md]: Isn't the writing "golang"?

File: _posts/2017/2017-08-01-going-forward.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+When we use our pipeline to integrate new parts (VM's, Server, even RPi's) into our network infrastructure, all the relevant data are put in to our CMDB.
+In this post we want to take a closer look towards the information flow and how we implemented it.
+The goal for us today is to see how the data goes into our CMDB, and how we use the data to create network related configurations.
+
+### Structure
+
+Before we take a look at how to get data into our CMDB we need some structure.
+By default, the CMDB has some preconfigured objects.
+One of these objects is called the Layer-3-Net.
+For those who forgot the [OSI model ](https://en.wikipedia.org/wiki/OSI_model), the Layer 3 abstraction is called the network Layer.
```
Comment (BirgitBader) [inline _posts/2017/2017-08-01-going-forward.md]: ... network layer.

File: _posts/2017/2017-08-01-going-forward.md [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+Every node in this network needs an IP/MAC address, and in most of our networks we have DHCP servers providing these IP's for specific MAC's.
+
```
Comment (BirgitBader) [inline _posts/2017/2017-08-01-going-forward.md]: remove blank line

**Comment by BirgitBader** [comment]
@cseeger-epages I have added the final review and exchanged the header image (now with free license). Please do a final check and give your ok, so that I can publish it on 1st of August. Thanks :)

**Comment by cseeger-epages** [comment]
looks good :+1:  :) as always great thanks to you @BirgitBader for doing so well when i was in vacation.

**Pull request merged**